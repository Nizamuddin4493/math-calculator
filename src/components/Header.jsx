import React from 'react';
import { Link } from 'react-router-dom';
import deleteCross from './images/delete-cross.png';
import home from './images/home.png';
import left from './images/left.png';
import right from './images/right.png';
import searchingBar3 from './images/searching-bar3.png';

function Header() {
  return (
    <>
      <nav>
        <div className="website-top">
          <div className="website-title">
            <h2 className="a-web-page">A Web Page</h2>
          </div>
          <div className="web-content">
            <img src={left} alt="left arrow" className="arrows" />
            <img src={right} alt="right arrow" className="arrows" />
            <img src={deleteCross} alt="delete" className="cross" />
            <img src={home} alt="home" className="home" />
            <div className="address-bar">
              <p className="address">https://</p>
            </div>
            <img src={searchingBar3} alt="search" className="search" />
          </div>
        </div>
        <div className="navigation">
          <div className="nav-title">
            <h2>Math Magicians</h2>
          </div>
          <div className="nav-links">
            <div className="link-container link-border">
              <h4 className="nav-link-item list-action">
                <Link to="/">Home</Link>
              </h4>
            </div>
            <div className="link-container link-border">
              <h4 className="nav-link-item right add-action">
                <Link to="/calDesign">Calculator</Link>
              </h4>
            </div>
            <div className="link-container">
              <h4 className="nav-link-item contact-action">
                <Link to="/quote">Quote</Link>
              </h4>
            </div>
          </div>
        </div>
      </nav>
    </>
  );
}

export default Header;
