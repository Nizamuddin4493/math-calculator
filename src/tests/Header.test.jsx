import { BrowserRouter as Router } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import Header from '../components/Header';

it('renders correctly', () => {
  const tree = TestRenderer.create(<Router><Header /></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
