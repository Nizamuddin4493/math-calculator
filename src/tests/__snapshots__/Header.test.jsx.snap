// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav>
  <div
    className="website-top"
  >
    <div
      className="website-title"
    >
      <h2
        className="a-web-page"
      >
        A Web Page
      </h2>
    </div>
    <div
      className="web-content"
    >
      <img
        alt="left arrow"
        className="arrows"
        src="left.png"
      />
      <img
        alt="right arrow"
        className="arrows"
        src="right.png"
      />
      <img
        alt="delete"
        className="cross"
        src="delete-cross.png"
      />
      <img
        alt="home"
        className="home"
        src="home.png"
      />
      <div
        className="address-bar"
      >
        <p
          className="address"
        >
          https://
        </p>
      </div>
      <img
        alt="search"
        className="search"
        src="searching-bar3.png"
      />
    </div>
  </div>
  <div
    className="navigation"
  >
    <div
      className="nav-title"
    >
      <h2>
        Math Magicians
      </h2>
    </div>
    <div
      className="nav-links"
    >
      <div
        className="link-container link-border"
      >
        <h4
          className="nav-link-item list-action"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </h4>
      </div>
      <div
        className="link-container link-border"
      >
        <h4
          className="nav-link-item right add-action"
        >
          <a
            href="/calDesign"
            onClick={[Function]}
          >
            Calculator
          </a>
        </h4>
      </div>
      <div
        className="link-container"
      >
        <h4
          className="nav-link-item contact-action"
        >
          <a
            href="/quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </h4>
      </div>
    </div>
  </div>
</nav>
`;
